docs/status.md

```diff
@@ -27,21 +27,21 @@ one of the following tracking labels.
 
```

Comment:
The description of the PR seems out of date. E.g. "Change in column "Type Inference" for all_reduce op -> yes(need-revisit)" doesn't elaborate on why the "Type Inference" column has changed for other ops.

Reply:
"why the "Type Inference" column has changed for other ops"
There's no other changes in column "Type Inference" other than all_reduce op. Please let me know if the diff page shows diffidently on you side. I update (rebase) the PR, but no more updates on the column "Type Inference".